src/azure-cli/azure/cli/command_modules/appconfig/tests/latest/test_appconfig_commands.py

```diff
@@ -431,7 +431,8 @@ def test_resolve_keyvault(self, key_vault, resource_group):
                  checks=[self.check('[0].key', secret_name),
                          self.check('[0].value', secret_value)])
 
-        exported_file_path = 'export_keyvault.json'
+        exported_file_path = os.path.join(TEST_DIR, 'export_keyvault.json')
```

Comment:
hi @shenmuxiaosen is this file `export_keyvault.sjon` deleted by intention?

Reply:
@yungezz Yes, we let them make this change. The deleted file was incorrectly located in the root folder of the project.